.github/workflows/verify-examples.yml

```diff
@@ -53,6 +53,12 @@ jobs:
         echo SKAFFOLD_BINARY=$PWD/out/skaffold >> $GITHUB_ENV
         out/skaffold config set --global collect-metrics false
 
+    - name: Set up Java17
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
```

Comment:
I think we can remove the `distribution` text here (not sure if this is related to why it's failing)